notes/admin.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+from .models import Note, PersonalNote
+
+class NoteAdmin(admin.ModelAdmin):
+    readonly_fields = ('created_at', 'last_modified')
+
+# Register your models here.
+admin.site.register(Note, NoteAdmin)
```

Comment:
put Note and PersonalNote in a tuple `admin.site.register((Note, PersonalNote), NoteAdmin)`